xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-ui/src/main/resources/XWiki/Lightbox/LightboxAdministration.xml

```diff
@@ -0,0 +1,349 @@
+<?xml version="1.1" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.4" reference="XWiki.Lightbox.LightboxAdministration" locale="">
+  <web>XWiki.Lightbox</web>
+  <name>LightboxAdministration</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>LightboxAdministration</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{velocity output="false"}}
+## To avoid mixing wiki syntax with html, unwrap the result of the xproperty displayer.
+#macro (stripHTMLMacro $displayOutput)
+  $stringtool.removeEnd($stringtool.removeStart($displayOutput, '{{html clean="false" wiki="false"}}'), '{{/html}}')
+#end
+#macro (showProperty $propertyName)
+  &lt;dl&gt;
+    &lt;dt&gt;
+      #set ($propertyClass = $xclass.get($propertyName))
+      &lt;label#if ($editing) for="${lightboxConfigClassName}_0_${propertyName}"#end&gt;
+        $escapetool.xml($services.localization.render("image.lightbox.admin.${propertyName}.label"))
```

Comment:
Why not use instead the translated property pretty name ``translatedPrettyName``?